lib/howitzer/web/iframe_dsl.rb

```diff
@@ -72,7 +72,8 @@ module ClassMethods
 
```

Comment:
Extend code documentation with more examples.
Use  # @raise [NameError] if page class can not be found